R/data.table.R

```diff
@@ -968,7 +968,39 @@ replace_dot_alias = function(e) {
         }
         # fix for long standing FR/bug, #495 and #484
         allcols = c(names_x, xdotprefix, names_i, idotprefix)
-        if ( length(non_sdvars <- setdiff(intersect(av, allcols), c(bynames, ansvars))) ) {
+        non_sdvars <- setdiff(intersect(av, allcols), c(bynames, ansvars))
+        
+        # added 'mget' - fix for #994
+        if (any(c("get", "mget") %chin% av)){
+          # get('varname') is too difficult to detect which columns are used in general
+          # eval(macro) column names are detected via the  if jsub[[1]]==eval switch earlier above.
+          b_warn = FALSE 
```

Comment:
nitpick: i'm not understanding this variable name, what's it mean?

Reply:
From my VBA days, b = boolean. Some habits are hard to get rid of. See also [Hungarian Notation](https://en.wikipedia.org/wiki/Hungarian_notation). 

Regarding the variable, I removed it.